charts/prometheus-pushgateway/Chart.yaml

```
@@ -1,22 +1,22 @@
 apiVersion: v2
-appVersion: "v1.6.0"
+appVersion: "v1.6.1"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 2.4.0
+version: 2.5.0
```

The change is really a patch.

```suggestion
version: 2.4.1
```